src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -161,14 +180,28 @@ export class StudentProfilePageComponent implements OnInit {
     this.ngbModal.open(confirmDeleteProfilePicture);
   }
 
+  /**
+   * Gets the profile picture as blob image.
+   */
+  getProfilePicture(): Observable<Blob> {
+    return this.httpRequestService.get(this.profilePictureEndPoint, {}, 'blob');
```

Comment:
I think a better way is to create a service and encapsulate the API call. Maybe just `student-profile-service`

Reply:
I realize there is already  a `student-profile.service.ts` file under `services`. Should I create this method under the service?

Reply:
Yup.